packages/core/src/execution-engine/workflow-execute.ts

```diff
@@ -1153,8 +1174,10 @@ export class WorkflowExecute {
 			}
 			// For trigger nodes in any mode except "manual" do we simply pass the data through
 			return { data: inputData.main as INodeExecutionData[][] };
-		} else if (nodeType.webhook) {
-			// For webhook nodes always simply pass the data through
+		} else if (nodeType.webhook && !nodeType.description.requestDefaults) {
+			// Check if the node have requestDefaults(RoutingNode),
+			// else for webhook nodes always simply pass the data through
+			// as webhook method would be called by WebhookService
```

Comment:
I don't understand what this addition does. Why is it needed?

Reply:
`sendAndWait` operation require two methods `execute` and `webhook`

it causes issues with routing nodes since `webhook` clause precised routing node clause
if no `customOperation` defined for resource+operation and there is `webhook` method node would return input data

that's why we need to check here whether it's not routing node

Reply:
So did I understand correctly that we are checking if the node type is a declarative node (executed using `RoutingNode`), we don't want to go down this branch? If that is the case, can we extract the check into a separate variable? E.g. something like

```ts
const isDeclarativeNode = ...
...

} else if (nodeType.webhook && !isDeclarativeNode) {...}
```

That would convey the meaning what we are doing here much more clearly